### PR TITLE
Show date of last activity for each student

### PR DESCRIPTION
**Pull request by martinzugnoni**
Solves https://github.com/rmotr/rmotr-sis/issues/25

As a profesor at rmotr.com I want to see in SIS the date of the last activity in the system for each student.

last-activity

### CODE REVIEW

File: rmotr_sis/accounts/middlewares.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+
+
+class LastActivityMiddleware(object):
+
+    def process_request(self, request):
+
+        # NOTE: This could cause performance issues in the future, but considering
+        # the current traffic rates we have right now it shouldn't be a problem.
+        if request.user.is_authenticated():
```
Comment (santiagobasulto) [inline rmotr_sis/accounts/middlewares.py]: What if we just do it for current students?
```
if student:
    log_activity()
```

**Comment by santiagobasulto** [comment]
+1

File: rmotr_sis/accounts/middlewares.py (hunk 1 above)
Comment (martinzugnoni) [inline rmotr_sis/accounts/middlewares.py]: It would be also interesting to see if any ex-student keeps entering or performing actions in SIS.

Comment (santiagobasulto) [inline rmotr_sis/accounts/middlewares.py]: You're right. +1

**Pull request merged**